expression/builtin_string_vec_test.go

```diff
@@ -166,7 +166,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.BitLength: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString}},
 	},
-	ast.FindInSet: {},
+	ast.FindInSet: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}},
```

Comment:
can we add some testcase to cover strlist is a empty string and str is in the strlist?

Reply:
I added.is it ok?

Reply:
we can use `constStrGener` to generate a given str

Reply:
![image](https://user-images.githubusercontent.com/32595964/68763045-70eebb00-0652-11ea-905d-78438afee301.png)
Is it right?

Reply:
right, but a bit too much, I think two is enough,
and we can keep this one
`{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}},`